.travis.yml

```diff
@@ -41,11 +41,9 @@ matrix:
     xcode_destination: platform=iOS Simulator,OS=12.2,name=iPhone X
     env: NODE_VERSION=10.15.3
 
-    branches:
-      only:
-      - master
```

Comment:
The idea under this line was to make PR build only if PR is targeted to master. IMO it's a reasonable approach not to build every commit pushed to any branch in the repo.

Reply:
Fixed.